### DISCUSSION TITLE
Returning the documentType alias back to the hover when editing

### DISCUSSION THREAD

**Discussion by tomotrepo** [umbraco/Umbraco-CMS#14201] [Features and ideas]
I would like you to return the documentType alias in edit mode when hovering the documentType name. This was in version 7, but now it's gone. Otherwise, it has no meaning for editors, but it makes it a lot easier for me as a programmer, because sometimes the documentType has a different displayed name than the alias, and then I always have to go back to settings.

Regards

**Comment by nathanwoulfe** [comment]
Hey @tomotrepo, are you meaning in the info app on a content node? The document type alias is available when the user has access to the settings section and the application is running in debug:

https://github.com/umbraco/Umbraco-CMS/blob/f3668e0aabee6c4ff7c3ed0dea55fd2421a79fc4/src/Umbraco.Web.UI.Client/src/common/directives/components/umbnodepreview.directive.js#L107C1-L113

![image](https://user-images.githubusercontent.com/3248070/236071405-4b4dac54-4978-4a5b-9897-9fbd5c1a3a5a.png)

**Reply by tomotrepo** [reply]
Hi,

I meant specifically this hover:

![Capture](https://user-images.githubusercontent.com/7202849/236243422-c34721fc-1bfc-44c0-a517-8fef21d4ec15.PNG)

A practical example: when programming a page, I have a lot of these elements and sometimes I forget how I defined the alias. And then later, when I have to fix something, I also don't remember all the aliases and then have to click in the settings and back. As I said, for editors it doesn't matter, but for programmers it's a nice tool, and if it used to be there, I'm not sure I know why it got removed.

Regards

**Reply by nathanwoulfe** [reply]
Yup, same logic applies here - debugging enabled, user has access to settings section. 

Looking at the Git history, this has been the case since July 2020. It was part of a larger change around converting directives to components, and I'm assuming was done to avoid data leaks in production.

**Reply by tomotrepo** [reply]
...was done to avoid data leaks in production.

All clear. Thnx.

**Reply by nathanwoulfe** [reply]
That's an assumption on my part - it's not particularly sensitive information, but really only useful in a development environment.
